rules/scripts.mk

```diff
@@ -23,11 +23,15 @@ $(SYSCTL_NET_CONFIG)_PATH = files/image_config/sysctl
 UPDATE_CHASSISDB_CONFIG_SCRIPT = update_chassisdb_config
 $(UPDATE_CHASSISDB_CONFIG_SCRIPT)_PATH = files/scripts
 
+SWSS_VARS_TEMPLATE = swss_vars.j2
+$(SWSS_VARS_TEMPLATE)_PATH = files/build_templates
```

Comment:
Instead of copying the file to `files/build_templates`, we can instead create symlink in the new dockers requiring it in the source code base and let the regular docker creation process copy the file.

Reply:
The builder has a staging directory when building the docker. As the symlink only contains the relative path of in the source code base, it cannot find the linked file in build.

Reply:
agree with @shi-su